labgrid/util/version.py

```diff
@@ -0,0 +1,15 @@
+"""
+This module contains helper functions for working with version.
+"""
+
+
+def labgrid_version():
+    from pkg_resources import get_distribution, DistributionNotFound
+
+    __version__ = "unknown"
```

Comment:
Using `__version__` isn't useful in a function, as it only has special semantics at the module scope.